raiden-ts/src/transfers/epics/secret.ts

```diff
@@ -418,7 +419,12 @@ export const transferAutoRegisterEpic = (
         takeUntil(grouped$.pipe(ignoreElements(), endWith(1))),
       ),
     ),
-    takeIf(config$.pipe(pluck('caps', Capabilities.NO_RECEIVE)), true),
+    takeIf(
+      config$.pipe(
+        pluck('caps'),
+        map((caps) => getCap(caps, Capabilities.RECEIVE)),
+      ),
+    ),
```

Comment:
Is there a negation missing?

Reply:
The 2nd param for this `takeIf` operator is `negate`, and is missing now.